gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/api/use_case/ImportCRDUseCase.java

```diff
@@ -431,13 +431,13 @@ private void deleteRemovedPages(Map<String, PageCRD> pages, String apiId) {
         }
     }
 
-    private void createOrUpdatePages(Map<String, PageCRD> pageCrds, String apiId, AuditInfo auditInfo) {
-        if (pageCrds == null || pageCrds.isEmpty()) {
+    private void createOrUpdatePages(Map<String, PageCRD> pageCRDs, String apiId, AuditInfo auditInfo) {
+        if (pageCRDs == null || pageCRDs.isEmpty()) {
             return;
         }
 
         var now = Date.from(TimeProvider.now().toInstant());
-        List<Page> pages = pageCrds.values().stream().map(this::initPageFromCRD).toList();
+        List<Page> pages = pageCRDs.values().stream().map(PageModelFactory::fromCRDSpec).toList();
```

Comment:
As var is a bit everywhere in the new package structure, I would go with it for the sake of consistency.